client/py/synnax/framer/client.py

```diff
@@ -127,11 +127,13 @@ def open_iterator(
         self,
         tr: TimeRange,
         params: ChannelParams,
+        chunk_size: int = 5e5,
```

Comment:
We should reduce this. Could make it easy to overload the maximum message size for certain websockret connections